### ISSUE TITLE
Support for .NET Core 3.1

### ISSUE THREAD

**Issue by ctearpak** [bugsnag/bugsnag-dotnet#122]
Since updating projects to ASP.NET Core 3.1, building throws hundreds of AD0001 warnings:

`Warning AD0001: Analyzer 'Microsoft.AspNetCore.Mvc.Analyzers.TopLevelParameterNameAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object'. (AD0001)`

This is almost certainly due to the dependency on all of the Microsoft.AspNetCore.* libraries that are at 2.2.0.

Bugsnag seems to be working OK, but having hundreds of warnings is not great.

Is there any way to get a build that plays nice with .NET Core 3.1? I noticed new builds haven't been submitted for over a year, and want to make sure us .NET Core folks are still supported.

**Comment by ctearpak** [comment]
So, I pulled the Bugsnag code down to make new projects that compiled on .NET Core 3.1.1, and I was still getting these issues. Turns out, I had a common library project that had a NuGet reference to Microsoft.AspNetCore.MVC, that wasn't actually needed by anything. Apparently, that was the issue, and the current build of the Bugsnag libraries looks fine.

It might still not be a bad idea to have a build specifically for .NET Core 3.1.
